contracts/ColonyTask.sol

```diff
@@ -283,26 +276,23 @@ contract ColonyTask is ColonyStorage, DSMath {
   }
 
   function setTaskDomain(uint256 _id, uint256 _domainId) public
+  self()
```

Comment:
Here we extend the authorisation to need not just the manager but the worker too. Is this something the product team have requested? Same question for `setSkill` function authorisation requirements

Reply:
Ok, we can restrict `setTaskDomain` and `setSkill` just to managers 👍